inframanager/api_handler/api_handler.go

```diff
@@ -660,37 +659,48 @@ func (s *ApiServer) AddDelSnatPrefix(ctx context.Context, in *proto.AddDelSnatPr
 
 func (s *ApiServer) NatTranslationDelete(ctx context.Context, in *proto.NatTranslation) (*proto.Reply, error) {
 	logger := log.WithField("func", "NatTranslationDelete")
-	logger.Infof("Incoming NatTranslationDelete %+v", in)
 
 	out := &proto.Reply{
 		Successful: true,
 	}
-	/*
-		service := store.Service{
-			ClusterIp: in.Endpoint.Ipv4Addr,
-			Port:      in.Endpoint.Port,
-			Proto:     in.Proto,
-		}
 
-		server := NewApiServer()
+	/* Currently supporting services only for the dpdk target */
```

Comment:
Did we miss uncommenting this part when the same was done for NatTranslationAdd call?

Reply:
Why is this check for tap even required? From Kubernetes perspective, NATTranslation has nothing to do with interface. Interface is more a function of configuration in Agent map file.